operator/pkg/gateway-api/tlsroute.go

```diff
@@ -87,19 +120,27 @@ func (r *tlsRouteReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		return err
 	}
 
-	return ctrl.NewControllerManagedBy(mgr).
+	builder := ctrl.NewControllerManagedBy(mgr).
 		// Watch for changes to TLSRoute
 		For(&gatewayv1alpha2.TLSRoute{}).
 		// Watch for changes to Backend services
 		Watches(&corev1.Service{}, r.enqueueRequestForBackendService()).
 		// Watch for changes to Reference Grants
 		Watches(&gatewayv1beta1.ReferenceGrant{}, r.enqueueRequestForReferenceGrant()).
+		// Watch for changes to Backend Service Imports
+		Watches(&mcsapiv1alpha1.ServiceImport{}, r.enqueueRequestForBackendServiceImport()).
```

Comment:
@MrFreezeex this is a leftover of an "unguarded" `ServiceImport` watch that let's the operator fail if deployed on a cluster without MCS installed (-> Gateway API conformance tests failed)

-> we can delete this line as this watch is covered below

Reply:
Ohhhh yes indeed, thanks :pray:. I was just getting started on looking why the test fails but it seems that you beat me at it :D.

Reply:
:smile: thanks for the fix! let's re-run the tests